### ISSUE TITLE
sinogram

### ISSUE THREAD

**Issue by RedhouaneBJM** [RI-imaging/FDTD_sinogram#1]
Hello,

i'm doing the same simulation as you except for the part "phantoms_meep" where i'm using python instead of C++.
I have several fields acquired with different angles, and i would like to know how did you do to calculate the sinogram. are you recording two fields (with and without phantom) for each angle ?.

Thank you

**Comment by paulmueller** [comment]
Yes, exactly. I divide the field with phantom by the field without phantom (background correction).

**Comment by RedhouaneBJM** [comment]
after that the divsion with/without how did you calculate the sinogram ?

**Comment by paulmueller** [comment]
You can find that code here: https://github.com/RI-imaging/FDTD_sinogram/blob/master/meep_tomo/meep_tomo/extract.py
